app/services/mods.ts

```diff
@@ -46,14 +46,28 @@ export function cleanup(mod: Mod, tempManifestPath: string) {
     pathsToPreserve = mod.pathsToPreserve;
   }
 
-  deleteFolderExcept(
-    mod.modPath,
-    mod.uniqueName === 'Alek.OWML'
-      ? ['Mods', 'OWML.Config.json', 'OWML.Manifest.json']
-      : ['config.json', 'save.json', 'manifest.json'].concat(
-          pathsToPreserve ?? []
-        )
-  );
+  let pathsToKeep;
+
+  if (mod.uniqueName === 'Alek.OWML') {
+    pathsToKeep = ['Mods', 'OWML.Config.json', 'OWML.Manifest.json'];
+  } else if (mod.uniqueName === 'bbepis.BepInEx') {
+    pathsToKeep = [
+      'BepInEx',
+      'OuterWilds_Alpha_1_2_Data',
+      'doorstop_config.ini',
+      'OuterWilds_Alpha_1_2.exe',
+      'winhttp.dll',
+      'BepInEx.Manifest.json',
```

Comment:
A bit confused about this. Seems like youre saying to just not delete anything if the mod being deleted is BepInEx. I understand this is because BepInEx is in the project root, but you essentially listed all the files here? Might as well just not run the delete function, no?

We have this `deleteFolderExcept` function because some mod updates will have files being deleted from one update to the next. So we can't just overwrite the mod files, we need to delete the old ones too. But, some mods also need to keep some user config files inside the mod dirs. So we have this exceptions list.

If a future BepInEx update drops `doorstop_config.ini` for instance, you would want it to be deleted, no? Then it shouldn't be in this list. Make sure you understand how this works, let me know if you have any questions.

Reply:
I changed `BepInEx` to `BepInEx/plugins/` but it deletes the mods anyways.